### PR TITLE
Show off Fast SAM

### PR DESCRIPTION
**Pull request by igor-davidyuk**
This tutorial features conversion and inference of the [Fast SAM model ](https://github.com/CASIA-IVA-Lab/FastSAM)

CVS-121987

### CODE REVIEW

**Comment by igor-davidyuk** [comment]
<p>removed </p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1372/discussion/#comment-1766837070'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by igor-davidyuk** [comment]
<p>changed, thanks</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1372/discussion/#comment-1766837097'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by igor-davidyuk** [comment]
> Why is putting a point on the image taking more time than actual model inference? Like 4s vs 391ms.

If you run the notebook on a remote server you may experience input lug due to gradio internal event processor

**Comment by apaniukov** [comment]
> > Why is putting a point on the image taking more time than actual model inference? Like 4s vs 391ms.
> 
> If you run the notebook on a remote server you may experience input lag due to gradio internal event processor

It is, yes.

**Review by apaniukov — APPROVED** [review]

File: notebooks/259-fast-segment-anything/259-fast-segment-anything.ipynb
Comment (aleksandr-mokrov) [inline notebooks/259-fast-segment-anything/259-fast-segment-anything.ipynb:1]: `The demo app below is created using`
Add a description what the demo does and how to use `Pixel selector type`

Comment (aleksandr-mokrov) [inline notebooks/259-fast-segment-anything/259-fast-segment-anything.ipynb:1]: Change `withContours=True` to `with_contours=True`

Comment (aleksandr-mokrov) [inline notebooks/259-fast-segment-anything/259-fast-segment-anything.ipynb:1]: `device = "cpu"` is not used

[the remark this answers is not in the document]
Comment (igor-davidyuk) [reply notebooks/259-fast-segment-anything/259-fast-segment-anything.ipynb:1]: fixed

[the remark this answers is not in the document]
Comment (igor-davidyuk) [reply notebooks/259-fast-segment-anything/259-fast-segment-anything.ipynb:1]: fixed

Comment (igor-davidyuk) [reply notebooks/259-fast-segment-anything/259-fast-segment-anything.ipynb:1]: added description

Comment (igor-davidyuk) [reply notebooks/259-fast-segment-anything/259-fast-segment-anything.ipynb:1]: fixed

Comment (igor-davidyuk) [reply notebooks/259-fast-segment-anything/259-fast-segment-anything.ipynb:1]: removed

File: notebooks/260-fast-segment-anything/260-fast-segment-anything.ipynb
Comment (aleksandr-mokrov) [inline notebooks/260-fast-segment-anything/260-fast-segment-anything.ipynb:1]: `
    results = model(image,
                    device="cpu",
                    retina_masks=use_retina,
                    iou=iou_threshold,
                    conf=conf_threshold,
                    imgsz=input_size,)
`
change `device="cpu"` to `device=DEVICE.value`

**Review by aleksandr-mokrov — APPROVED** [review]

File: notebooks/260-fast-segment-anything/260-fast-segment-anything.ipynb
Comment (igor-davidyuk) [reply notebooks/260-fast-segment-anything/260-fast-segment-anything.ipynb:1]: done

**Comment by igor-davidyuk** [comment]
<p>done</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1372/discussion/#comment-1774483462'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by igor-davidyuk** [comment]
<p>unfortunately, there is now a color plotting option, I increased the confidence threshold, see if it is better now</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1372/discussion/#comment-1774483477'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by eaidova — APPROVED** [review]

**Pull request merged**